src/git-extract

```diff
@@ -54,17 +54,18 @@ function preconditions {
 }
 
 
-function steps {
-  echo_if_true "stash_open_changes" "$INITIAL_OPEN_CHANGES"
+function should_stash_open_changes {
```

Comment:
How do you feel about making `true` the default option? I think this would be the less intrusive change.